pyFV3/initialization/__init__.py

```diff
@@ -1 +1,3 @@
 from .analytic_init import init_analytic_state
```

Comment:
Like in `stencils` this might be a good place for some cursory scientific description